EIPS/eip-3675.md

```diff
@@ -201,7 +206,6 @@ This event was removed for two reasons:
 
 The value of `FORK_NEXT` refers to the block number of the next fork a given node knows about and `0` otherwise.
```

Comment:
Is this section still needed? It seems to be properly described in the specification now.